### DISCUSSION TITLE
Content service to update the DIAL context - Part 2

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Knowlg/Community#64] [General Discussions]
Hi @vinukumar-vs 

Can you please provide more information on below ticket which are part of 5.0 Sunbird-Knowlg release.

what is the impact on existing dial codes, do we need to migrate existing dial codes?

https://project-sunbird.atlassian.net/browse/SB-30118

cc : @anandvarada @kameshbhr @mmanimegalai

**Comment by Jayaprakash8887** [comment]
https://project-sunbird.atlassian.net/wiki/spaces/SBDES/pages/3214016513/Adoption+of+Sunbird+DIAL+Context

No impact on existing dial codes. Migration will be necessary only if the adopter wants to update the existing DIAL code context information in DIAL service.
